### DISCUSSION TITLE
KDL Twitter account

### DISCUSSION THREAD

**Discussion by msikma** [kdl-org/kdl#203] [Ideas]
A totally random thought I had, but I thought it'd be nice to have an official KDL Twitter account for major updates or maybe to occasionally see some stuff people are building with it. I'd follow that.

**Comment by zkat** [comment]
do JSON and YAML and XML have these? I already have so many twitter accounts to maintain T_T

**Reply by hkolbeck** [reply]
IMO it's been fine just getting updates from your personal.

**Comment by msikma** [comment]
Well yeah, JSON and YAML and XML having an advocacy account would be ridiculous. They already have widespread adoption, everybody knows them, they've been completely figured out and battle tested.

KDL has none of those things. It's a very new language, 1.0.0 spec, and so for it to be used in a serious or mission critical project you need a stronger argument in favor of it. And it's nice to have a place to link to when you want to show people how awesome it is and how it's solving problems, and preferably it shouldn't be just the maintainer's personal account if it has a ton of other things on it as well. It can be a place where people send stuff to that they've made using it, to show that it has an edge over established technologies. It's really not so weird or uncommon—Nim, Rust, Zig and a ton of other newer languages all have Twitter accounts.

Anyway, I get that you're not interested in doing it, and it's not like the project _needs_ it. That's fine! It was just a random thought that I decided to suggest. But apparently it's considered such a laughable suggestion that now I'm sorry I ever said anything.

**Reply by zkat** [reply]
it's not laughable! I was asking a genuine question because I didn't know. I don't think it's a silly question to ask or a bad suggestion. I think it's ok to think we don't need it without making you feel like Less Than <3

Don't worry about it. No one thinks less of you!
